ci/scripts/run-shellcheck.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+EXCLUDED_DIRS="-name third_party -o -name vendor -o -name lowrisc_misc-linters"
+
+# Get an array of all shell scripts to check
+readarray -t shell_scripts < \
+    <(find . \( $EXCLUDED_DIRS \) -prune -o -name '*.sh' -print)
```

Comment:
Should we be searching `.` or `"$(dirname "$(readlink -e "${BASH_SOURCE[0]}")")"`?

Reply:
Yes - you're right. This would assume that the script is run from $REPO_TOP. I've modified this to `cd` into `${BASH_SOURCE[0]}/../..` which would be $REPO_TOP before running `find`.